src/pathfinder/npf/queue.cpp

```diff
@@ -287,7 +287,7 @@ void Hash::PrintStatistics() const
 	uint usage[200];
```

Comment:
```suggestion
	uint usage[200] = {};
```

(then remove the std::fill)

Reply:
And probably fix the declaration of `i` as well